src/elements/gameForm/gameForm.tsx

```diff
@@ -19,34 +19,36 @@ interface GameFormProps {
 }
 
 export default function GameForm({ children, game, onSubmitHandler }: GameFormProps) {
+  const initModel = {
+    ...game,
+    category: game?.genre,
+    minAge: game?.minAge.toString(),
+    pc: game?.platforms.includes("pc"),
+    ps5: game?.platforms.includes("ps5"),
+    xbox: game?.platforms.includes("xbox"),
+  };
   return (
-    <Form className={styles.form} onSubmit={onSubmitHandler}>
+    <Form initModel={initModel} className={styles.form} onSubmit={onSubmitHandler}>
       <h3 className={styles.subheadings}>Information</h3>
       <ControlWrapper text="Name">
```

Comment:
You have repetitive logic: controlWrapper together with control. Need to add inline `label for` by default into base react control. So you don't need to wrap it again and again.
Actual: `<ControlWrapper>  <TextControl> ....`
Expected: `<TextControl>...` (reuse render part from ControlWrapper directly in Base Control